package.json

```diff
@@ -109,6 +109,7 @@
   },
   "dependencies": {
     "balanced-match": "^2.0.0",
+    "colord": "^2.9.1",
```

Comment:
Changed to use colord package instead of reference file.

Reply:
The [`colord`](https://github.com/omgovich/colord) package looks nice to me. Just in case, could you tell us why you choose this package, please?

Reply:
[colord](https://www.npmjs.com/package/colord) seems to be used in [cssnano](https://www.npmjs.com/package/cssnano) and I think we can trust it. Also, another well-known package, [color](https://www.npmjs.com/package/color), doesn't seem to be able to parse modern syntax (e.g. `rgb(0 0 0)`), so I think colord is better.

Please let me know if you know any other better packages.